heuristic/alns.py

```diff
@@ -474,8 +480,8 @@ def calculate_objective(self, state, destroy, repair):
             self.saturdays,
             self.L_C_D,
             self.days,
-            self.competencies,
```

Comment:
Aren't competencies used when calculating objective function?

Reply:
Nope! At least not in `delta_calculations` 
![bilde](https://user-images.githubusercontent.com/1371159/81821832-a5283300-9532-11ea-8176-2e2e5d5011cb.png)